File: src/main/java/com/wafflestudio/draft/dto/request/AuthenticationRequest.java
package com.wafflestudio.draft.dto.request;

import lombok.Data;
import org.springframework.lang.Nullable;

@Data
public class AuthenticationRequest {
    private String grantType;

    @Nullable
    private String accessToken;
    @Nullable
    private String authProvider;
///
    @Nullable
    private String email;
    @Nullable
    private String password;
}

File: src/main/kotlin/com/wafflestudio/draft/dto/request/AuthenticationRequest.kt
package com.wafflestudio.draft.dto.request

import org.springframework.lang.Nullable

open class AuthenticationRequest(
        open var grantType: String? = null,

        @get:Nullable
        open var accessToken: String? = null,

        @get:Nullable
        open var authProvider: String? = null,

        ///
        @get:Nullable
        open var email: String? = null,

        @get:Nullable
        open var password: String? = null
)
